tests/zeroconfig/windows/testdata/resource_traces/aspnetcore.yaml

```diff
@@ -2,7 +2,6 @@ resource_spans:
 - attributes:
     deployment.environment: zc-iis-test
     host.name: <ANY>
-    os.type: windows
```

Comment:
Why is this changed? Is this what was set by the collector, and now we send data directly?

Reply:
The collector receiving the signals from the instrumentation is not using the default agent configuration anymore, the stripped down config used now doesn't have the `resourcedetection` processor that adds that attribute.